src/phases/tpch/normalized/Q1.yml

```diff
@@ -0,0 +1,55 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-query"
+Description: |
+  Run TPC-H query 1 against the normalized schema. Using an 'executionStats' explain causes each command to run its execution plan until no
+  documents remain, which ensures that the query executes in its entirety.
+
+batchSize: &batchSize 101  # The default batch size.
+query1Delta: &query1Delta -90
+
+TPCHNormalizedQuery1:
+  Repeat: 1
+  Database: tpch
+  Operations:
+  - OperationMetricsName: Query1
+    OperationName: RunCommand
+    OperationCommand:
+      explain:
+        aggregate: lineitem
+        pipeline:
+          [
+            {$match: {$expr: {$lte: ["$l_shipdate", {$dateAdd: {startDate: {$dateFromString: {dateString: "1998-12-01"}}, unit: "day", amount: {^Parameter: {Name: "Query1Delta", Default: *query1Delta}}}}]}}},
```

Comment:
Should the normalized query here be identical to the query in the jstest [here](https://github.com/10gen/mongo/pull/2061/files#diff-ce54cc54db72ff4e2b0cf7b85042ba4565ec425b64e514a2508f7dc5f0d1487d)? I see there is slight difference.

Reply:
Is there a difference other than the parametrization?

Reply:
Oh it's just a minor one where you use `$dateAdd` here but `$dateSubtract` in the jstest IIRC

Reply:
I'll update the integration test to match this.